status.c

```diff
@@ -882,7 +894,7 @@ status_prompt_translate_key(struct client *c, key_code key, key_code *new_key)
 		*new_key = 'B'|KEYC_VI;
 		return (1);
 	case 'd':
-		*new_key = '\025';
+		*new_key = '\025'; /* C-u */
```

Comment:
Other key escapes sequences in the code are annotated with the corresponding semantic key name in a comment, how about following that best practice here too?